tests/Cms/Files/FileTest.php

```diff
@@ -738,8 +757,42 @@ public function testPreviewUrl()
 		$this->assertSame('/test/test.pdf', $file->previewUrl());
 	}
 
+	public function testPreviewUrlUnauthenticated()
+	{
+		$page = new Page([
+			'slug'  => 'test',
+			'files' => [
+				[
+					'filename' => 'test.pdf'
+				]
+			]
+		]);
+
+		$file = $page->file('test.pdf');
+		$this->assertSame('/media/pages/test/' . $file->mediaHash() . '/test.pdf', $file->previewUrl());
```

Comment:
Shouldn't this return null in case of an unauthenticated user?

Reply:
Or do you think this current state  (also before PR) is actually a bug?

Reply:
I saw this here for the Page and I think it actually makes sense that the preview URL is null when the user is not authenticated. https://github.com/getkirby/kirby/blob/b4c0dfdd95fcc8556963d9fc2dbcb3cbd72a2b98/tests/Cms/Pages/PageTest.php#L607

Reply:
Changed it, but now it's a breaking change, so we would have to move this to v5?

Reply:
I would argue that this has been a pretty clear bug before. I'd say we could still put it in 4.4 but if you feel better to move it to 5, I'm fine with that as well.

Reply:
Ok, then ready for review/approval.